config/serverless.security.yml

```diff
@@ -22,3 +22,11 @@ xpack.serverless.plugin.developer.projectSwitcher.currentType: 'security'
 
 # Specify in telemetry the project type
 telemetry.labels.serverless: security
+
+# Search session disable for discover embedding
+data.search.sessions.enabled: false
```

Comment:
I think this is not necessary, because this is disabled in serverless anyways `config/serverless.yml`

Reply:
Thank you @Dosant , I was not aware of it.